CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -11,6 +11,17 @@ if vrPurchasePromptsEnabled then
 	return --Don't enable this file if the VR Purchase Prompts flag is ON
 end
 
+local success, result = pcall(function() return settings():GetFFlag('UsePurchasePromptLocalization') end)
+local FFlagUsePurchasePromptLocalization = success and result
+local function LocalizedGetString(key, rtv)
+	local success = pcall(function()
+		local LocalizationService = game:GetService("LocalizationService")
+		local CorescriptLocalization = LocalizationService:GetCorescriptLocalizations()[1]
```

Comment:
Someday this may be a value you get from localizationservice.
LocalizationService.CoreGUILocalizationIndex